lib/benchmarking/moab_enumeration_testing.rb

```diff
@@ -0,0 +1,110 @@
+# rails runner moab_enumeration_testing.rb
+require 'benchmark'
+require 'logger'
+
+##
+# simple class for collecting stats on how fast different moab listing methods run.
+#
+# Usage:
+# $ bundle exec rails console # from shell prompt
+# > require './lib/benchmarking/moab_enumeration_testing.rb'
+# > benchmarker = MoabEnumerationBenchmarker.new
+# > benchmarker.benchmark
+class MoabEnumerationBenchmarker
+  attr_accessor :storage_dirs, :all_results, :methods_to_test
+
+  def benchmarked_revision
+    pcc_app_home = '/opt/app/pcc/preservation_core_catalog'
+    revisions_log = "#{pcc_app_home}/revisions.log"
+
+    # last line of revisions.log if it exists
+    # or else grab current revision
+    if File.file?(revisions_log)
+      File.readlines(revisions_log).last
+    else
+      commit = `git rev-parse HEAD`
+      "at revision #{commit}"
+    end
+  end
+
+  def logger
+    @logger ||= Logger.new('log/benchmark.log')
```

Comment:
Any chance we could use a slightly more descriptive name here, i.e. a separate log file for this benchmarker?